manuscript/chapter3-contributor-mode.md

```diff
@@ -130,7 +166,15 @@ The big issue with chats is to stay on topic, and get back to the topic when the
 
 An example of this is when one piles many questions/issues in one single message, when people join after some time the discussion,they are unable to catch up from the very top sometimes. In that case is better to face one thing at the time and pass on to the next when the previous is exhausted and in doing that there is the need to be intransigent with the discussion remaining on topic.
 
-E-mails allow for recap, or a single notification for a group of mails, chat systems send a notification for every and each message that gets to be posted, this resulting in a dancing and ringing smartphone in less than a minute.
+E-mails allow for recap, or a single notification for a group of mails, chat systems send a notification for every and each message that gets to be posted, this resulting in a dancing and ringing smartphone in less than a minute.  
+
+For this reasons a chat solution is fine for some stuff and not for everything. Also understand the kind of technology for chatting is important, is not just a flow of messages like IRC (that anyway is getting abandoned for various reasons), but include usability, multi-device usage and various levels of anonymity.  
+For a lot of people use IRC or Jabber is difficult because they have no idea of what means use a "client", that need to be configured but also what pick for that scope based on their system. Often these clients don't have modern features like the link preview with an image, title of the pages etc, bot with buttons to press instead of commands etc.  
```

Comment:
For a lot of people used to use IRC or Jabber, it is difficult because

Reply:
This line is quite confusing. What do you want to express?

Reply:
the approach of this system is to download a client to configure to connect to a server and often this can be confusing as today we use to have app of a specific service.

Reply:
> When asking people to use IRC or Jabber, they are confused to have to configure a „client” based on their system.
> In today's world, people use apps which do those things for them.
> Plus, these clients don't have modern features like …